### PR TITLE
Collector's app deployment

### PR DESCRIPTION
**Pull request by shirmoran**

### CODE REVIEW

File: k8s/collector-deployment.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    metadata:
+      labels:
+        app: collector
+    spec:
+      containers:
+      - name: collector-container
+        image: quay.io/testnetworkfunction/collector:latest
+        env:
+          - name: DB_CONN_STR
+            value: "collectoruser:password@tcp(mysql.default.svc.cluster.local:3306)/"
```
Comment (bnshr) [inline k8s/collector-deployment.yml]: Only DB_URL and DB_PORT should be hard coded here.
The rest like DB_USER, DB_PASSWORD should be inside the secret.
Sample code is mentioned here.

apiVersion: v1
kind: Secret
metadata:
  name: collector-db-creds
type: Opaque
data:
  mysql-username: bnNzZnVzcg==
  mysql-password: bnNzZnBhc3N3ZA==
  mysql-db-name: bnNzZmRi

**Review by bnshr — CHANGES_REQUESTED** [review]

File: k8s/collector-deployment.yml (hunk 1 above)
Comment (bnshr) [reply k8s/collector-deployment.yml]: All the values should be base64 encoded and from the collector app, you have to access by decoding. 
And in the collector deployment you get the env variable values mounting the secret.

Comment (shirmoran) [reply k8s/collector-deployment.yml]: changed

**Review by bnshr — APPROVED** [review]
Lgtm!

File: k8s/collector-deployment.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: collector-db-creds
+  namespace: default
+type: Opaque
+data:
+  mysql-username: Y29sbGVjdG9ydXNlcg==
```
Comment (sebrandon1) [inline k8s/collector-deployment.yml:8]: We might have to come up with a better solution to checking in these values and regenerate the creds for the "real" DB.

**Review by sebrandon1 — APPROVED** [review]

**Pull request merged**